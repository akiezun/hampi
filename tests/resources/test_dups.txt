S = "x" | "x" ;
A = | | "foo" | B | D | D D ;
B = "a" | "b" | "a" | "b";
C = ("a" "b")? | ("b")* | ("a" "b")? | ("b")? | ( ("a" "b")? )+;
D = "a" | "b" | "a" | "b";