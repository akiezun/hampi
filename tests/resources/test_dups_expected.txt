A =  | "foo"  | B  | B B  ;
B = "a"  | "b"  ;
C = ("a" "b")?  | ("b")*  | ("b")?  | (("a" "b")?)+  ;
S = "x"  ;
